src/hooks/useIsMobile.ts

```diff
@@ -14,7 +14,7 @@ declare global {
 }
 
 const useIsMobile = () => {
-  const [isMobile, setIsMobile] = useState(true)
+  const [isMobile, setIsMobile] = useState(false)
```

Comment:
We use this hook in places to avoid loading heavy assets, like videos. By initializing to `false` the video fetching wil be triggered. That's why it was initialized to `true`. With this change we do not benefit from smaller assets loading on mobile.